tests/app/views/contexts/conftest.py

```diff
@@ -197,19 +197,26 @@ def response_metadata():
 @pytest.fixture
 def fake_session_data():
     return SessionData(
-        tx_id="tx_id",
-        schema_name="some_schema_name",
-        period_str="period_str",
         language_code=None,
-        launch_language_code=None,
-        schema_url=None,
-        ru_name="ru_name",
-        ru_ref="ru_ref",
-        response_id="response_id",
-        case_id="case_id",
     )
 
 
+@pytest.fixture
+def fake_questionnaire_store_metadata():
+    return {
+        "tx_id": "tx_id",
+        "schema_name": "some_schema_name",
+        "period_str": "period_str",
+        "language_code": None,
+        "launch_language_code": None,
```

Comment:
`launch_language_cod` is not a valid metadata, that key only existed in session data.
This is the issue with just using an arbitrary dictionary to store this data. I was going to suggest we need a data model like session data, but given we need that for the new launch/reception pattern I am happy to leave the work for that.

Reply:
Still outstanding.

Reply:
Removed now